endpoints/openrtb2/auction.go

```diff
@@ -787,6 +797,21 @@ func setSiteImplicitly(httpReq *http.Request, bidReq *openrtb.BidRequest) {
 			}
 		}
 	}
+	if bidReq.Site != nil {
+		setAmpExt(bidReq.Site, "0")
+	}
+}
+
+func setAmpExt(site *openrtb.Site, value string) {
```

Comment:
Any particular reason you put this in auction.go rather than amp_auction.go?

Reply:
No reason... but that makes sense. I'll move it.

It's used in both files... started out named something else, but with "amp" in the name I can definitely see how it fits better in the `amp_auction` file.